test/acceptance/features/steps/workercluster.py

```diff
@@ -476,3 +487,13 @@ def ensure_secret_key_has_the_right_value(context, cluster_name: str, secret_nam
         target=lambda: primaza_cluster.read_secret_resource_data(namespace, secret_name, key) == bytes(value, 'utf-8'),
         step=1,
         timeout=30)
+
+
+@step(u'On Worker Cluster "{cluster_name}", the service binding "{service_binding_name}" in namespace "{namespace}" is bound')
+def ensure_service_binding_secret_is_bound(context, cluster_name: str, service_binding_name: str, namespace: str):
+    worker_cluster = context.cluster_provider.get_worker_cluster(cluster_name)
+    group = "primaza.io"
+    version = "v1alpha1"
+    plural = "servicebindings"
+    response = worker_cluster.read_custom_resource_status(group, version, plural, service_binding_name, namespace)
+    assert response["status"]["state"] == "Ready"
```

Comment:
we already have a very similar step in `kubernetesapi.py`. I'd say we can reuse this step. 

https://github.com/primaza/primaza/blob/346cadb510f35f8e824474df21fea1323add721f/test/acceptance/features/steps/kubernetesapi.py#L181

Reply:
Done